### PR TITLE
[8] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by Lolatech07**
## Task: `Add Page Navigation to your pages`
- `Task submission
Screenshot1
Screenshot2

`

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
This is not what you are ask to do

**Comment by adaugochi** [comment]
You ask to create navigation links on your product page and contact form page

**Comment by Lolatech07** [comment]
I see. My oversight. I didn't understand the task as you explained.

So navigation link for Product page to be on the contact page and vice
versa?

**Comment by Lolatech07** [comment]
Hello. Could you kindly respond?

Thanks.

**Comment by Lolatech07** [comment]
> You ask to create navigation links on your product page and contact form page



> You ask to create navigation links on your product page and contact form page

Hi. Please review updates to my PR. Thanks.

**Comment by Lolatech07** [comment]
Hi. Please review changes to my PR. Thanks.

File: submissions/lolaOluwadahunsi/htmlForms/ecommerce_website.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/lolaOluwadahunsi/htmlForms/ecommerce_website.html:1]: This file already exist in your html folder. You should remove it from here. @Lolatech07

File: submissions/lolaOluwadahunsi/htmlForms/contact_form_page.html [hunk 2]
```diff
@@ -5,6 +5,12 @@
     <title>Contact Form</title>
 </head>
 <body>
+<nav>
+    <ul>
+        <li><a href="ecommerce_website.html">Products</a></li>
```
Comment (ToluAkin) [inline submissions/lolaOluwadahunsi/htmlForms/contact_form_page.html:10]: Link to the page in your html folder

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Lolatech07 You need to link your ecommerce page with contact form.

**Comment by Lolatech07** [comment]
Hi Tolu.

I thought I already linked the ecommerce and contact pages.

If you please clarify what I need to do here.

I've gone through additional resources as well and I'm not sure what else
needs to be done.

Thanks.

**Comment by ToluAkin** [comment]
@Lolatech07 This should help []( https://www.w3schools.com/Html/html_filepaths.asp)

**Pull request closed without merging**